### ISSUE TITLE
Launcher didn't open. And "'∩╗┐' is not recognized"

### ISSUE THREAD

**Issue by AeroElectro** [crocokyle/Valorant-Ultrawide-Fix#2]
Launcher didn't open at first. I had to manually change the line in ValorantLauncher.bat to correct path which for me is "C:\Games\Riot Games\Riot Client\..."

Can you confirm if there is anything else that needs to be customized?

Also, after the above change, cmd pops up and stays stating:
 C:\Users\xxx\Documents\ValorantUltrawideHack>∩╗┐
'∩╗┐' is not recognized as an internal or external command,
operable program or batch file.

**Comment by crocokyle** [comment]
Thanks, I'm just now realizing the encoding for the launcher should have been `OEM` rather than `UTF8`. 

I updated the release so you should be able to download and try again now. I'll leave this open for now to make sure it worked.

**Comment by AeroElectro** [comment]
The "not recognized" text is gone.

But now defender shows this message while installing:
"Microsoft Defender SmartScreen prevented an unrecognized app from starting. Running this app might put your PC at risk."
I had to bypass defender by clicking "More info" then "run anyways".

**Also, not sure if known issue but I'll leave below:**
1. Launcher still didn't open until manually edited the path (not a big deal but might need documentation).
2. CMD window has to be closed manually.
3. The new shortcut has no pink icon that is mentioned in readme.

**Comment by crocokyle** [comment]
The Microsoft Defender warning is most likely because a UAC prompt was added to the PowerShell script. I think now that I have solved the permissions issues, that is no longer required. I will do some testing and possibly remove that prompt if it isn't necessary. I'll be working on this issue in #3 

- Good point on the launcher path. This needs to be dynamic. Same issue with the shortcut icon, it isn't a dynamic path to the exe. I opened a new issue for these here: #5 
- CMD window closing automatically is fixed now
